gstreamer-mpegts-sys/Cargo.toml

```diff
@@ -25,13 +25,6 @@ shell-words = "0.1.0"
 tempdir = "0.3"
 
 [features]
-v1_2 = []
-v1_4 = ["v1_2"]
-v1_6 = ["v1_4"]
-v1_8 = ["v1_6"]
-v1_10 = ["v1_8"]
-v1_12 = ["v1_10"]
-v1_14 = ["v1_12"]
```

Comment:
This should stay

Reply:
I was a bit worried about these not being there but assumed it was due to a change in gstreamer-plugins-bad. However it seems another facet of the Meson build as opposed to the Autotools build.

Reply:
That's missing features in gir. Just revert that part